backend/consultations/serializers.py

```
@@ -24,7 +24,7 @@ class Meta:
         model = Consultation
         fields = [
             'id', 'consultation_type', 'name', 'email', 'phone',
-            'password', 'message', 'files', 'uploaded_files', 
+            'password', 'message', 'status', 'files', 'uploaded_files', 
             'created_at', 'updated_at'
         ]
         read_only_fields = ['id', 'files', 'created_at', 'updated_at']
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)